CodeStarterCamp_Week2/main.swift

```diff
@@ -1,43 +1,44 @@
 //
-//  main.swift
+//  File.swift
 //  CodeStarterCamp_Week2
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by 예강이다 on 2022/11/11.
 //
 
 import Foundation
 
-// Mark:- 로또 번호 생성
-func createLottoNumbers() -> Set<Int> {
+// Mark: - 로또 번호 생성
+func makeLottoNumbers() -> Set<Int> {
     var lottoNumbers: Set<Int> = []
     
     while lottoNumbers.count < 6 {
         lottoNumbers.insert((Int.random(in: 1...45)))
     }
-    
     return lottoNumbers
 }
 
-// Mark:- 로또번호와 내 번호 비교
-func compareLottoNumbers(lottoNumbers: Set<Int>, myLottoNumbers: [Int]) -> [Int] {
-    let compareNumbers = lottoNumbers.intersection(myLottoNumbers)
-
-    return Array(compareNumbers).sorted()
+// Makr: - 회차 번호 생성
+func makeLottoByRounds() -> Dictionary<String, [Int]> {
+    var lottoDictionary: Dictionary<String, [Int]> = [String: [Int]]()
+    
+    for round in 1...5 {
+        let lottoNumbersArray = Array(makeLottoNumbers())
+        let round = "\(round)"
+        lottoDictionary[round] = lottoNumbersArray
```

Comment:
```suggestion
        lottoDictionary["\(round)"] = lottoNumbersArray
```
상수 선언 없이 바로 사용하는게 깔끔할 것 같네요 👀

Reply:
훨씬 보기 깔끔한거 같아서 수정 했습니다!